azext_edge/edge/params.py

```diff
@@ -787,13 +787,13 @@ def load_iotops_arguments(self, _):
         context.argument(
             "subject_name",
             options_list=["--subject-name", "--sn"],
-            help="The subject name string embedded in the application instance certificate."
+            help="The subject name string embedded in the application instance certificate. "
```

Comment:
Explain the purpose e.g. "When provided the input will be validated against the value embedded in the certificate."